apps/src/templates/ContainedLevel.jsx

```diff
@@ -31,7 +33,37 @@ const ContainedLevel = React.createClass({
       codeStudioLevels.lockContainedLevelAnswers();
     } else {
       // No answers yet, disable Run button until there is an answer
-      $('#runButton').prop('disabled', true);
+      let runButton = $('#runButton');
+      runButton.prop('disabled', true);
+      // Disabled buttons don't trigger mouse events, add a dummy element to
+      // receive the click and trigger the display of the callout.
+      let clickReceiver = $('<div id="clickReceiver"/>');
+      let boundingClientRect = runButton.get(0).getBoundingClientRect();
+      clickReceiver.css({
+        height: boundingClientRect.height + 'px',
+        width: boundingClientRect.width + 'px',
+        position: 'absolute',
+        top: 0,
+      });
+      $('#gameButtons').append(clickReceiver);
+      clickReceiver.bind('click',
+        () => $(window).trigger('attemptedRunButtonClick'));
+
+      addCallouts([{
```

Comment:
I wonder if this makes sense to do elsewhere in contained level init?  It's a bit weird to have this as a side effect of mounting the component.

If I unmount and re-mount the component multiple times (e.g. in Storybook) do I get multiple callouts?